packages/lesswrong/components/ultraFeed/UltraFeedSuggestedUserCard.tsx

```diff
@@ -143,6 +144,7 @@ const UltraFeedSuggestedUserCard = ({
   const classes = useStyles(styles);
   const { openDialog } = useDialog();
   const followButtonRef = useRef<HTMLDivElement>(null);
+  const currentUser = useCurrentUser();
```

Comment:
If this is just to check the user's logged in state, prefer `useCurrentUserId`, which doesn't cause rerenders if anything about the current user changes